tests/download.spec.ts

```diff
@@ -272,8 +272,8 @@ it.describe('download event', () => {
     await page.close();
   });
 
-  it('should report new window downloads', async ({browser, server, browserName, headful}) => {
-    it.fixme(browserName === 'chromium' && headful);
+  it('should report new window downloads', async ({browser, server, browserName, headful, browserChannel}) => {
+    it.fixme(browserName === 'chromium' && headful && !!browserChannel, 'Disabled until upstream fix hits stable');
```

Comment:
Instead of disabling it for everything, you can enable it conditionally on a version number. See `interception.spec.ts`:
```js
it.fail(isChromium && +browserVersion.split('.')[0] < 91);
```

Reply:
Removed this filter, since the test passes with current stable.